packages/server/test/e2e/2_cdp_spec.ts

```diff
@@ -0,0 +1,21 @@
+const e2e = require('../support/helpers/e2e')
+const Fixtures = require('../support/helpers/fixtures')
+
+describe('e2e cdp', function () {
+  e2e.setup()
+
+  it('fails when remote debugging port cannot be connected to', function () {
+    return e2e.exec(this, {
+      project: Fixtures.projectPath('remote-debugging-port-removed'),
+      spec: 'spec.ts',
+      browser: 'chrome',
+      expectedExitCode: 1,
+      snapshot: true,
+      onStdout: (stdout: string) => {
+        return stdout
```

Comment:
instead of doing all this, just accept `CYPRESS_REMOTE_DEBUGGING_PORT` env var and let that be the override to `getPort()` in `chrome.coffee` so we can return a static value and not have to massage it out.